syntax = "proto3";

import "google/protobuf/timestamp.proto";

message Location {
  int32 personId = 1;
  int32 latitude = 2;
  int32 longitude = 3;
  google.protobuf.Timestamp createdTime = 4;
}

message Status { string currentState = 1; }

service LocationService {
  rpc updateLocation(Location) returns (Status) {}
}